include/util/uuid.h

```diff
@@ -0,0 +1,6 @@
+#ifndef UTIL_UUID_H
+#define UTIL_UUID_H
+
+int generate_uuid(char out[static 37]);
```

Comment:
For consistency with the rest of wlroots, we could change this to return a `bool`.